modules/rgbd/include/opencv2/rgbd/kinfu.hpp

```diff
@@ -16,6 +16,65 @@ namespace kinfu {
 //! @addtogroup kinect_fusion
 //! @{
 
+struct CV_EXPORTS_W VolumeParams
+{
+    enum VolumeKind
+    {
+        TSDF = 0,
+        HASHTSDF = 1,
+        COLOREDTSDF = 2
+    };
+
+    /** @brief Kind of Volume
+        Values can be TSDF (single volume) or HASHTSDF (hashtable of volume units)
+    */
+    CV_PROP_RW int kind;
+
+    /** @brief Resolution of voxel space
+        Number of voxels in each dimension.
+        Applicable only for TSDF Volume.
+        HashTSDF volume only supports equal resolution in all three dimensions
+    */
+    CV_PROP_RW int resolutionX;
+    CV_PROP_RW int resolutionY;
+    CV_PROP_RW int resolutionZ;
+
+    /** @brief Resolution of volumeUnit in voxel space
+        Number of voxels in each dimension for volumeUnit
+        Applicable only for hashTSDF.
+    */
+    CV_PROP_RW int unitResolution = { 0 };
```

Comment:
just `= 0;`

---

other parameters should be initialized too (by default ctor, see below existing code).

Reply:
done